bin/reth/src/node/mod.rs

```diff
@@ -21,41 +25,67 @@ use reth_network::{
     error::NetworkError,
     NetworkConfig, NetworkHandle, NetworkManager,
 };
-use reth_primitives::{hex_literal::hex, Account, Header, H256};
+use reth_primitives::{Account, Header, H256};
 use reth_provider::{db_provider::ProviderImpl, BlockProvider, HeaderProvider};
 use reth_stages::stages::{bodies::BodyStage, headers::HeaderStage, senders::SendersStage};
-use std::{
-    net::SocketAddr,
-    path::{Path, PathBuf},
-    sync::Arc,
-};
+use std::{net::SocketAddr, path::Path, sync::Arc};
 use tracing::{debug, info};
 
-// TODO: Move this out somewhere
-const MAINNET_GENESIS: &str = include_str!("../../res/chainspec/mainnet.json");
-
 /// Start the client
 #[derive(Debug, Parser)]
 pub struct Command {
     /// The path to the database folder.
-    // TODO: This should use dirs-next
-    #[arg(long, value_name = "PATH", default_value = "~/.reth/db", value_parser = parse_path)]
-    db: PathBuf,
+    ///
+    /// Defaults to the OS-specific data directory:
+    ///
+    /// - Linux: `$XDG_DATA_HOME/reth/db` or `$HOME/.local/share/reth/db`
+    /// - Windows: `{FOLDERID_RoamingAppData}/reth/db`
+    /// - macOS: `$HOME/Library/Application Support/reth/db`
+    #[arg(long, value_name = "PATH", verbatim_doc_comment, default_value_t)]
+    db: DbPath,
+
+    /// The path to the configuration file to use.
+    #[arg(long, value_name = "FILE", verbatim_doc_comment, default_value_t)]
+    config: ConfigPath,
+
+    /// The chain this node is running.
+    ///
+    /// Possible values are either a built-in chain or the path to a chain specification file.
+    ///
+    /// Built-in chains:
+    /// - mainnet
+    /// - goerli
+    /// - sepolia
+    #[arg(
+        long,
+        value_name = "CHAIN_OR_PATH",
+        verbatim_doc_comment,
+        default_value = "mainnet",
+        value_parser = chain_spec_value_parser
+    )]
+    chain: ChainSpecification,
```

Comment:
Note here: This does not work yet because of `ForkId` stuff, i.e. if you pass `--chain sepolia` it will never sync